src/app/shared/actions/account-actions/account-actions.component.ts

```diff
@@ -14,19 +14,44 @@ import { Account } from '../../models/account.model';
 })
 export class AccountActionsComponent {
   @Input() public account: Account;
-  @Output() public onAccountChanged: EventEmitter<Account>;
+  @Output() public onAccountLock: EventEmitter<Account>;
```

Comment:
@Output() public onAccountLock = new EventEmitter<Account>();
do the same for others. 
remove it from constructor